src/views/List.jsx

```diff
@@ -20,12 +21,13 @@ export function List({ data, listPath }) {
 		//Debounce function
 		const getItem = setTimeout(() => {
 			//Filters through all items to return an array of items that match input
-			const filteredResults = data.filter((item) =>
+			let filteredResults = data.filter((item) =>
 				item.name
 					.toLowerCase()
 					.replace(/\s+/g, '')
 					.includes(searchTerm.toLowerCase().replace(/\s+/g, '')),
 			);
+			filteredResults = filteredResults.sort(comparePurchaseUrgency);
```

Comment:
just fyi, instead of doing a `let` and reassigning it, you can chain `sort` off of the `filter` and it would work the same

Reply:
Thank you Rachel, changed with chaining sort filter and sort method together to filteredResults, commit [3a8e78fef962b8f4d5835eabe9c0585e4c1352c3](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/pull/34/commits/3a8e78fef962b8f4d5835eabe9c0585e4c1352c3)